react/components/Textarea/index.js

```diff
@@ -10,6 +10,14 @@ class Textarea extends Component {
     }
   }
 
+  static CharacterCountdown = props => {
+    let classes = 'mid-gray f6 mt3 '
```

Comment:
I'd add a `margin-left` case the width of the whole element shrinks to the point that the helper text is right next to the count down like "This is some helper text blabla 23"